pkg/interfaces/contracts/vault/ICompositeLiquidityRouter.sol

```diff
@@ -30,7 +30,8 @@ interface ICompositeLiquidityRouter {
      * the "parent" pool, and also make sure limits are set properly.
      *
      * @param pool Address of the liquidity pool
-     * @param exactUnderlyingAmountsIn Exact amounts of underlying tokens in, sorted in token registration order of
+     * @param isWrappedToken An array indicating whether the input token is a wrapped or underlying token
+     * @param exactAmountsIn Exact amounts of underlying/wrapped tokens in, sorted in token registration order
      * wrapped tokens in the pool
```

Comment:
```suggestion
     * @param isWrappedToken An array indicating whether the input token is a wrapped or underlying token, 
     * sorted in token registration order of wrapped tokens in the pool
     * @param exactAmountsIn Exact amounts of underlying/wrapped tokens in, sorted in token registration order 
     * of wrapped tokens in the pool
```

Here and elsewhere. No matter if we're using the underlying or wrapped token, we're still receiving values in the order of the wrapped token registration order. And I'd suggest adding an order comment to the isWrappedToken array (maybe the same as exactAmountsIn "sorted in token registration...", or "same order as exactAmountsIn", or something).